scale/job/migrations/0045_auto_20180830_1812.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.15 on 2018-08-30 18:12
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('job', '0044_jobtypetag_job_type'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='jobexecution',
+            name='docker_image',
+            field=models.TextField(default=''),
+        ),
+        migrations.AddField(
+            model_name='jobtyperevision',
+            name='docker_image',
+            field=models.TextField(default='not-a-real-image'),
+        ),
+        migrations.AlterField(
+            model_name='jobtype',
+            name='docker_image',
+            field=models.CharField(default='', max_length=500),
```

Comment:
What was this field altered for?

Reply:
Without a default, all unit tests fail as its a nullable field and jobtyperevision is not.